src/lib/y2network/widgets/bond_port.rb

```diff
@@ -196,15 +196,15 @@ def justify_dev_name(name)
       # device names if at least two devices shared the same physical port id
       # TODO: backend method
       #
-      # @param slaves [Array<String>] bonding slaves
+      # @param ports [Array<String>] devices included in the bonding
       # @return [Hash{String => Array<String>}] of duplicated physical port ids
-      def repeated_physical_port_ids(slaves)
+      def repeated_physical_port_ids(ports)
```

Comment:
Here we see a previous usage of the term _port_. We should explain in a comment whether ports are the same as slaves or not.

Reply:
Extended method's documentation